src/odemis/acq/__init__.py

```diff
@@ -38,6 +38,7 @@
     ScannedRemoteTCStream, ScannedTCSettingsStream
 from odemis.util import img, fluo, executeAsyncTask
 import time
+import resource
```

Comment:
Not needed?

Reply:
We actually do need it (see l. 456). On ubuntu, there is a limit on the number of files that can be opened at the same time. This also seems to apply to VAs. This limit is by default 1024, which is not enough if we want to listen to all VAs, so the gui doesn't start and I get an error message. Typing `ulimit -n 2048` in the command line works and the python resource function does the same.

Reply:
I'm not really sure where to put it. Is the `Instantiator` in `modelgen` a good place? Or in `model.core`? What do you suggest?